deprecate.dd

```diff
@@ -10,6 +10,7 @@ $(SPEC_S Deprecated Features,
 
     $(TABLE2 Deprecated Features,
         $(THEAD Feature,                                                          Spec,  Dep,    Error,  Gone)
+        $(TROW $(DEPLINK Comma operator),                                         future, &nbsp;, &nbsp;, &nbsp;)
```

Comment:
"Using the result of a comma expression"
We can extend it later, but so far that's what we are discussing.